pkg/doc/verifiable/common.go

```diff
@@ -55,16 +55,44 @@ func (ja JWSAlgorithm) jose() (jose.SignatureAlgorithm, error) {
 // Proof defines embedded proof of Verifiable Credential
 type Proof interface{}
 
-type typedID struct {
+// TypedIDExtraFields is a map of extra fields which extend typed ID field.
+type TypedIDExtraFields map[string]interface{}
```

Comment:
Is there a reason that this is an extra type? instead of just making ExtraFields in TypedID a map[string]interface{} directly?

Reply:
It's just done to improve readability. No methods are defined for `ExtraFields`.